components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1564,7 +1564,7 @@ real(rtype) function polysvp1(T,i_type)
     else
 
        print*
-       print*,'** polysvp1 i_type must be 0 or 1 but is: ',i_type,T
+       print*,'** polysvp1 i_type must be 0 or 1 but is: ',i_type
```

Comment:
I actually think it is really beneficial to have the T printed here, I would advocate for putting it back in the same line so you get the i_type and the offending temp.